layer/nrlf/producer/fhir/r4/model.py

```diff
@@ -553,7 +553,7 @@ class DocumentReference(BaseModel):
         str,
         Field(
             description="The status of this document reference.",
-            pattern="[^\\s]+(\\s[^\\s]+)*",
+            pattern="^current$",
```

Comment:
As above, should be updated in the `api/producer/swagger.yaml` and then regenerated.